presets/4.3/vtx/NanoHawk.txt

```diff
@@ -0,0 +1,43 @@
+#$ TITLE: Emax NanoHawk
```

Comment:
Something like
Emax NanoHawk vtx SA2.1?

Reply:
#$ CATEGORY: VTX

Why would the title need to state it again?